ingest/src/commands/all.test.ts

```diff
@@ -0,0 +1,115 @@
+import { PageStore, EmbeddedContentStore, assertEnvVars } from "chat-core";
+import { INGEST_ENV_VARS } from "../IngestEnvVars";
+import { doAllCommand, makeIngestMetaStore } from "./all";
+import { MongoClient } from "mongodb";
+
+import "dotenv/config";
+
+jest.setTimeout(1000000);
+
+describe("allCommand", () => {
+  const { MONGODB_CONNECTION_URI: connectionUri } =
+    assertEnvVars(INGEST_ENV_VARS);
+
+  const mockEmbeddedContentStore: EmbeddedContentStore = {
+    async deleteEmbeddedContent() {
+      return;
+    },
+    async findNearestNeighbors() {
+      return [];
+    },
+    async loadEmbeddedContent() {
+      return [];
+    },
+    async updateEmbeddedContent() {
+      return;
+    },
+  };
+  const mockPageStore: PageStore = {
+    async loadPages() {
+      return [];
+    },
+    async updatePages() {
+      return;
+    },
+  };
+
+  let databaseName: string;
+
+  beforeEach(async () => {
+    databaseName = `test-all-command-${Date.now()}-${Math.floor(
+      Math.random() * 10000000
+    )}`;
```

Comment:
nit: seems redundant to have both Date.now() and Math.floor() calls. both of these would individually provide the randomness necessary for a unique DB name

Reply:
Because this is a unit test if both run within the same ms (not likely but theoretically possible) we would have a conflict. Better safe than mildly confused and annoyed by a flaky test.